src/normalizers/keywordProtector.js

```diff
@@ -0,0 +1,34 @@
+import { regexMatchingDecoderMaker } from './regex.js';
+
+export function keywordProtector (keywordMap) {
+    const encoder = regexMatchingEncoderMaker(
+        'keywordProtectorDecoder',
```

Comment:
probably not very important but my suggestion was to have a `keywordMapMaker(name, keywordMap)` signature so that the name of encoder/decoder is not hard-coded and can support multiple instances.

Reply:
I think of it as protecting from the translator, but your suggestion makes sense.  Changed.